### ISSUE TITLE
[enhancement] display image

### ISSUE THREAD

**Issue by greppp** [pirateradiohack/PiRadio#10]
Hi,

I was wondering how hard it would be, for, when we add an url, to also submit a square image, which would be displayed on the screen?

As an example :

**Mission Control Radio** on SomaFM

Radio information URL : https://somafm.com/missioncontrol/
Stream URL : https://ice5.somafm.com/missioncontrol-128-mp3
Image link : https://somafm.com/img3/missioncontrol-400.png

Thank you!

**Comment by pirateradiohack** [comment]
Yes, that would be very neat indeed.

My priority is to port the project to version 2, with support for Raspberry Pi Zero 2 W. So for the moment I will be doing this. But once it is done I can focus on this kind of features. 

In version 2 I stopped using the Raspberry Pi OS in favor of another project called [myMPDos](https://github.com/jcorporation/myMPDos). This project has some support for images when adding and playing a radio. I may use that as the basis for image display on the screen.  
I have to dive into it before I can confirm if I can do it.

**Comment by greppp** [comment]
Amazing, yes it looks definitely more suited to the task!

**Comment by greppp** [comment]
As a quick note, it would be nice to have the possibility to rotate the screen (in my application my pi zero will be installed vertically, which means I need to rotate the screen back 90 deg in software).
